src/platform/imx8ulp/include/platform/lib/mailbox.h

```diff
@@ -0,0 +1,70 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Peng Zhang
```

Comment:
Be consistent with the author name and mail address.
Keep it as in the other files.